templates/documentation/.github/workflows/test.yml

```diff
@@ -0,0 +1,8 @@
+name: Run unit tests
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test
+        run: exit 0
```

Comment:
Is this file only for debug?

Reply:
all target repos are supposed to have tests, for now documentation has tests that are always happy

Reply:
Could you add a comment about this?

Reply:
done